src/xunit.console.uwp/HostedAppxTest.cs

```diff
@@ -162,19 +163,20 @@ public void Run(bool debug, TimeSpan timeout)
             }
             else
             {
-                Console.WriteLine("No results found at  {0}, copying logs ", resultPath);
```

Comment:
We should always copy the logs, even if we have an XML file, since just having one doesn't mean it's correct.